src/cmake/GeosxOptions.cmake

```diff
@@ -4,6 +4,7 @@ message("CMAKE_SYSTEM_NAME = ${CMAKE_SYSTEM_NAME}")
 message("CMAKE_HOST_APPLE = ${CMAKE_HOST_APPLE}")
 
 ### OPTIONS ###
+option( GEOSX_ENABLE_FPE "" ON)
```

Comment:
@klevzoff @corbett5 @TotoGaz  It seems we could have more specificity to our cmake variables by prepending all of the GEOSX make variables with `GEOSX_`. Is there a reason we aren't doing this?

Reply:
I'm on board with that

Reply:
That would probably reduce conflicts, so let's take it (furthermore being consistent always helps).

Reply:
I don't think we want to prefix all of the CMake variables but we definitely should prefix our macros. For instance LvArray needs to detect if OpenMP or CUDA is enabled and it is not going to look at `GEOSX_ENABLE_OPENMP`. Plus I think BLT has some logic built around common flags like `ENABLE_OPENMP` and `ENABLE_TESTS` but I could be wrong.

RAJA doesn't have a prefix on most of their options.
https://github.com/LLNL/RAJA/blob/2f99cffb2d09713030e4040ae1a8b905e6eb55eb/CMakeLists.txt#L53